### ISSUE TITLE
NTR: methods of generating alleles in model organisms

### ISSUE THREAD

**Issue by sbello** [obi-ontology/obi#1396]
The Alliance of Genome Resources curators have developed a set of terms from describing the various generation methods used to create new mutant alleles in model organisms. We have a spreadsheet that lists:

- Term name
- proposed definition
- definition reference
- synonyms
- OBI parent term
- xrefs in other ontologies/vocabularies (primarily the FB vocabulary)

We would like to have these added to OBI for use in our unified allele curation.

spreadsheet link: https://docs.google.com/spreadsheets/d/1gHhap1RwICcjokdpcJ38g1FGfYASkKrWvj-DTzmq1Ds/edit?usp=sharing 

Please let us know if you have any questions or need modifications.

**Comment by handemcginty** [comment]
Discussion on Aug.2nd 2021 suggested that these terms may fit best under "planned process". Is that consistent with what the curators had in mind? We would love to hear from one of the curators give us more details during one of our next discussions for OBI.

**Comment by sbello** [comment]
Our proposed parents were all descendants of planned process but we had suggested using either random mutagenesis (OBI:0002620) or induced mutation (OBI:0001154) as the closest existing OBI parents. Several of the terms would have one of the new terms as the direct parent.

If you let me know when the next meeting is we can do our best to get one or more curators to join your call.

**Comment by handemcginty** [comment]
Thank you Sue for your prompt response, our next meeting is next Monday, but the agenda already seems pretty full, I will suggest the call on the 16th or another day, here is more information about the calls and links to the call etc. : https://docs.google.com/document/d/1eEutJAG56gncTsWf2sAqHa4a9pQAuCbhsg_kmbF78tw/edit#
Looking forward to the discussion.

**Comment by mgiglio99** [comment]
Hi Sue,
Would you be available to discuss your new terms during the OBI call next Monday the 23rd at noon Eastern?
Michelle

**Comment by sbello** [comment]
Yes, sorry I forgot about it today. I have another call at noon but I can
skip it for this. I've added the call on the 23rd to my calendar so I won't
forget.
Sue

**Comment by mbrochhausen** [comment]
Was discussed in OBI meeting August 23. General agreement that the terms looked good.

**Comment by sbello** [comment]
I don't have editor privileges on OBI that I am aware of so I'm not sure why this was assigned to me. Is there something that is needed before the terms can be added?

**Comment by jamesaoverton** [comment]
@sbello Do you have someone on your team who can edit the `src/ontology/obi-edit.owl` file in Protege, add these terms, and make a Pull Request? The [README](https://github.com/obi-ontology/obi/blob/master/README.md) has instructions but there is a learning curve.

**Comment by sbello** [comment]
I'm familiar with protege and pull requests so I can do that I just wasn't aware that was what you all wanted. I'll get those in as soon as I can.

**Comment by jamesaoverton** [comment]
Thanks @sbello. We use this Google Sheet to assign new OBI IDs: https://docs.google.com/spreadsheets/d/1tpDrSiO1DlEqkvZjrDSJrMm7OvH9GletljaR-SDeMTI/edit Please just request permissions and I'll grant them. If you have trouble with anything, let me know.

**Comment by sbello** [comment]
@jamesaoverton I have a branch ready to commit but I'm getting a message that I don't have the permissions for the obi repository. Who can help with this?
